specs.json

```
@@ -452,6 +452,10 @@
     "url": "https://webassembly.github.io/exception-handling/js-api/",
     "forkOf": "wasm-js-api-2"
   },
+  {
+    "url": "https://webassembly.github.io/gc/core/",
+    "forkOf": "wasm-core-1" 
```

```suggestion
    "forkOf": "wasm-core-1"
```
trailing white space triggers lint error